server/services/conditionsService.ts

```diff
@@ -5,8 +5,14 @@ import * as builder from './licenceWithConditionsBuilder'
 import { getAdditionalConditionsConfig, standardConditions, CURRENT_CONDITION_VERSION } from './config/conditionsConfig'
 import { AdditionalConditions, Licence } from '../data/licenceTypes'
 
-export default function createConditionsService() {
-  function getFullTextForApprovedConditions(licence: Licence) {
+export default class ConditionsService {
```

Comment:
moving to class so get better type info than a factory method returning a dynamic object